code/modules/admin/IsBanned.dm

```diff
@@ -26,19 +27,8 @@
 
 	var/datum/entity/player/P = get_player_from_key(ckey)
 
-	//check if the IP address is a known TOR node
-	if(CONFIG_GET(flag/ToRban) && ToRban_isbanned(address))
-		log_access("Failed Login: [src] - Banned: ToR")
-		message_admins("Failed Login: [src] - Banned: ToR")
-		return list("reason"="Using ToR", "desc"="\nReason: The network you are using to connect has been banned.\nIf you believe this is a mistake, please request help at [CONFIG_GET(string/banappeals)]")
-
-	// wait for database to be ready
 
 	. = P.check_ban(computer_id, address)
-	if(.)
-		return .
-
-	return ..() //default pager ban stuff
```

Comment:
Are you sure this is related and you do want to change?

Reply:
escaped from the atomisation chamber of #5628